### ISSUE TITLE
Disable Mirroring not working for redirections 

### ISSUE THREAD

**Issue by palash018** [responsively-org/responsively-app#1088]
# 🐞 bug report

### ✍️ Description
 Disable mirroring is not working well, On clicking buttons which doesn't redirect it works but when redirected links are clicked other device windows also redirects.

### 🔬 Minimal Reproduction
Follow Development setup from readme using yarn, and click on disable mirroring after running app. Click on about in the Google search bar in the disabled device.

### 🌍 Your Environment
Windows 11 x64 yarn 1.22.19

### 🔥 Exception or Error or Screenshot

https://github.com/responsively-org/responsively-app/assets/109908523/2ea2f846-d837-4978-a26c-0f12cc2735ca

**Comment by palash018** [comment]
Can i get this assigned?

**Comment by manojVivek** [comment]
@palash018 We stopped assigning issues, please go ahead and let us know if you get any questions along the way. .

**Comment by palash018** [comment]
@manojVivek were you able to reproduce this issue?

**Comment by haseeb196** [comment]
It could be because, when you click on a link within a device, it alters the main URL, causing each device to change. However, there exists only one primary URL that is employed across all devices, even if you click on the event mirroring button.

**Comment by palash018** [comment]
@haseeb196 yeah i checked it out. This seems to be how it's working , are you trying to solve this?

**Comment by negar-75** [comment]
Hi 
I noticed this issue as well check some factors.
don't you think this problem related to this function that does not work correctly?

onst toggleEventMirroring = async () => {
    if (webview == null) {
      console.log('webview == null');
      return;
    }
    try {
      await webview.executeJavaScript(
        `
        if(window.___browserSync___){
          window.___browserSync___.socket.${
            eventMirroringOff ? 'open' : 'close'
          }()
        }
        true
      `
      );

      setEventMirroringOff((pre) => !pre);
    } catch (error) {
      // eslint-disable-next-line no-console
      console.error('Error while toggleing event mirroring', error);
    }
  };
  may it be related to  ___browserSync___ or electron?

**Comment by Mehranmj4** [comment]
[![.github/workflows/azure-webapps-node.yml](https://github.com/Mehranmj4/Mj/actions/workflows/azure-webapps-node.yml/badge.svg)](https://github.com/Mehranmj4/Mj/actions/workflows/azure-webapps-node.yml)
